src/AddProjectsPage/ChosenTeam.tsx

```diff
@@ -10,13 +10,13 @@ class ChosenTeam extends React.Component<{
     if (team.length === 0) {
       chosenTeam = null;
     } else {
-      if (team.length === 1) {
-        chosenTeam = (
-          <div className="tag-container" key={1}>
+      chosenTeam = team.map((teamMemeber: string, index: number) => {
```

Comment:
Making this into its own function would improve the overall readability.

Reply:
moved into its own function